tests/mail/tests.py

```diff
@@ -1135,6 +1135,35 @@ def test_email_multi_alternatives_content_mimetype_none(self):
         with self.assertRaisesMessage(ValueError, msg):
             email_msg.attach_alternative("<p>content</p>", None)
 
+    def test_body_contains_successful_search(self):
+        email_msg = EmailMultiAlternatives()
+        email_msg.body = "I am content."
+        email_msg.attach_alternative("I am content.", "text/html")
+
+        self.assertTrue(email_msg.body_contains("I am content"))
```

Comment:
From our [coding style](https://docs.djangoproject.com/en/5.0/internals/contributing/writing-code/coding-style/)
> Use [assertIs(…, True/False)](https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertIs) for testing boolean values, rather than [assertTrue()](https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertTrue) and [assertFalse()](https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertFalse), so you can check the actual boolean value, not the truthiness of the expression.

Please update the asserts to `self.assertIs(.., True/False)`

Reply:
wow, didn't know about that 😱